REQUIRE

```diff
@@ -1,4 +1,4 @@
-julia 0.7.0-alpha
+julia 0.7-alpha
```

Comment:
Why is this necessary? Seems like 0.7.0-alpha should be sufficient, as it's the same as 0.7-alpha.

Reply:
Looks like neither works... :-/

Reply:
...welp.

Reply:
CC: @Keno

Reply:
0.7.0-alpha would have been fine. Looks like femtocleaner got itself stuck somewhere. I'll restart it for now.

Reply:
Try again (you can close and re-open the PR to re-trigger the check).

Reply:
Hmm, it's possible that this package itself causes it to hang. I'll take a look.

Reply:
Nope, there it is. It just takes some time.

Reply:
Thanks, it worked! It even finished in -1m :-)

Reply:
It tried to do the following:
```
  ## histograms ##
  function sturges(n)  # Sturges' formula
      n==0 && return one(n)
-     ceil(Integer, digits=log2(n))+1
+     ceil(Integer, digits=digits=log2(n))+1
  end
I think you can see where that'd go. I'll deploy a fix and report back.
```

Reply:
I have deployed https://github.com/JuliaComputing/Deprecations.jl/commit/42ca003101a57341046acd8f2bc3528d1e064cc3 which fixes that particular issue. If you hit the "Fix it" button again, you should be able to trigger femtocleaner to get you a fixed commit.